scripts/build_package.py

```diff
@@ -251,7 +251,7 @@ def handle_external_repository(dep, clean, env, verbose):
         if source:
             tmp = tempfile.mkdtemp()
             LOG.debug("Downloading font files " + source + " to " + tmp)
-            for fformat, agent in font_user_agents.iteritems():
+            for fformat, agent in font_user_agents.items():
```

Comment:
`items()` creates a copy in the iteration, which is, at this point, worthless memory overhead.

Take note that we are using **Python 2.7**, and **not** Python 3. The removal of `iteritems()` and the behaviour change of `items()` to only be a view is a Python 3 thing.